### ISSUE TITLE
[Content]: Big Query Destination Setup

### ISSUE THREAD

**Issue by jsabor** [segmentio/segment-docs#3754]
### What article on segment.com/docs is affected?

https://segment.com/docs/connections/storage/catalog/bigquery/

### What part(s) of the article would you like to see updated?

Under "Create Project and Enable Big Query," I would add the user needs to create a dataset in BQ that matches the Segment Source name to not result in a failure.  The way the documentation reads, it seems Segment will automatically write(create) the dataset for the user in BQ.

**Comment by markzegarelli** [comment]
Hi @jsabor, can you share a bit more about the issue you ran into?

I was able to follow the instructions to create a BQ instance without first creating the dataset.

**Comment by jsabor** [comment]
Hi @markzegarelli , thanks for reaching out and I found the error.  I created a Destination with the wrong BQ location (I used us-west-1b vs us-west-1) within Segment; however, this brought up an interesting workaround.  Before I made the fix in Segment, I created a dataset in my BQ project with the correct region and data was flowing regardless of the region I specified in Segment (and why I sent the content update request).  All in all, confirming docs are correct and I stumbled on a weird quirk.
